content/spin/v2/key-value-store-tutorial.md

```diff
@@ -403,12 +403,13 @@ func main() {}
 
 ## Building and Deploying Your Spin Application
```

Comment:
Do you want to change this title to reflect the change below?

Reply:
Great point. Thanks @itowlson 
I have updated it.